pkgs/tools/networking/p2p/libtorrent-jesec/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "libtorrent-jesec";
+  version = "0.13.8-r1";
+
+  src = fetchFromGitHub {
+    owner = "jesec";
+    repo = "libtorrent";
+    rev = "v${version}";
+    sha256 = "sha256-Eh5pMkSe9uO0dPRWDg2BbbRxxuvX9FM2/OReq/61ojc=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ openssl zlib ];
+
```

Comment:
I tried
```
  checkInputs = [ gtest ];
  
  doCheck = true;
  
  preBuild = ''
    export LD_LIBRARY_PATH=build
  '';
```
to make the tests work. Do you have an idea?

Reply:
I'm not sure either, I tried a few other things with no luck.

@jesec Do you happen to have any idea why the libtorrent test binary would be failing to load the libtorrent shared object?

Reply:
@winterqt You should pase the output so that @jesec doesn't have to set up Nix. But I think an upstream issue is better.

Reply:
@winterqt How about you file an upstream issue, and add
```
checkInputs = [ gtest ];

# https://github.com/jesec/libtorrent/issues/...
doCheck = false;
```
?

Reply:
@dotlambda Sure, done. Apologies for the wait.